### PR TITLE
fix typeerror

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

**Comment by nickpell** [comment]
https://sentry.io/dimagi/commcarehq/issues/336205165/

**Comment by nickpell** [comment]
✅

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1522,7 +1522,7 @@ def send_dimagi_ending_reminder_email(self):
         context = {
             'domain': domain,
             'end_date': end_date,
-            'client_reminder_email_date': end_date - datetime.timedelta(days=30),
+            'client_reminder_email_date': (self.end_date - datetime.timedelta(days=30)).strftime(USER_DATE_FORMAT),
```
Comment (dannyroberts) [inline corehq/apps/accounting/models.py]: Do you want to put this into the user's timezone? Or is `end_date` just a date and not a datetime?

**Review by dannyroberts — APPROVED** [review]

File: corehq/apps/accounting/models.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/accounting/models.py]: just a date

**Pull request merged**